force-app/main/default/classes/RecurrenceRule_TEST.cls

```diff
@@ -84,6 +84,50 @@ public with sharing class RecurrenceRule_TEST {
         System.assertEquals(days, rule.getDays());
     }
 
+    @IsTest
+    private static void shouldSetAndGetBySetPos() {
+        Integer bySetPos = -1;
```

Comment:
Can we call this defaultBySetPos in the test to make it clear we're testing a default? Took me a sec to remember why there was no `.with`... here. :)